### DISCUSSION TITLE
Get-PnPAzureADGroup limiting groups returned

### DISCUSSION THREAD

**Discussion by londify** [pnp/powershell#3070] [General]
Hi,

I upgraded from 1.2 to 2.1.1 and noticed that the Get-PnPAzureADGroup command is limiting the groups i get back to 100.  Before I was getting 7k+.  Anyone else experiencing this?

**Comment by ronaldmavarez** [comment]
I am creating an issue based on this, I am facing the same problem after updating to 2.1.1
